### ISSUE TITLE
Query Result, Edit Inline not working

### ISSUE THREAD

**Issue by gaby64** [phpmyadmin/phpmyadmin#18824]
### Describe the bug

Edit Inline is not working, clicking the button does nothing.

### To Reproduce

Go to a table, click edit inline in the result_query card 

### Expected behavior

The input to appear.

### Server configuration

Ubuntu 23.10
Nginx 1.25.3
MAriaDB: 11.0.3-MariaDB-1:11.0.3+maria~ubu2210-log
PHP version: 8.2.11
PHPMyAdmin: 6.0.0-dev Git revision: [3a7baa4]

### Client configuration

Google Chrome
Windows 11

**Comment by kamil-tekiela** [comment]
Can you show a screenshot? Do you have unique results? i.e. a primary key or unique key?

**Comment by gaby64** [comment]
Nothing special, just click on any table

![image](https://github.com/phpmyadmin/phpmyadmin/assets/2108555/f885afe4-85b2-4f94-83aa-5da0b6fee603)

**Comment by kamil-tekiela** [comment]
Yeah, it works fine. Please provide an MCVE. You can also try it on https://demo.phpmyadmin.net/master/public/index.php?route=/sql&pos=0&db=sakila&table=actor

**Comment by gaby64** [comment]
I have installed from the latest snapshot file, it works.

Any reason git pull breaks things vs the snapshot?

```
cd /usr/share/phpmyadmin
sudo git stash
sudo git pull
sudo git stash pop
sudo composer update --no-dev
sudo yarn install --production
```

running this, phpmyadmin has all kinds of issues in the web interface

**Comment by kamil-tekiela** [comment]
Did you reload browser cache, i.e. Ctrl + F5?

**Comment by gaby64** [comment]
> Did you reload browser cache, i.e. Ctrl + F5?

I did

**Comment by williamdes** [comment]
Hi @gaby64 
Can you try with the last commit of 6.0 ?

You can also use the [(phpMyAdmin 6.0+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_6.0+snapshot) also available as a [non official docker image](https://github.com/sudo-bot/docker-phpmyadmin-snapshots#readme).
